Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2679 @@
+        output.append(reservation)
+
+    raw = json.loads(json.dumps(output, cls=DatetimeEncoder))
+    ec = {'AWS.EC2.ReservedInstances(val.ReservedInstancesId === obj.ReservedInstancesId)': raw}
+    human_readable = tableToMarkdown('AWS EC2 Reserved Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def monitor_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    data = []
+    response = client.monitor_instances(InstanceIds=parse_resource_ids(args.get('instancesIds')))
+
+    for instance in response['InstanceMonitorings']:
+        data.append({
+            'InstanceId': instance['InstanceId'],
+            'MonitoringState': instance['Monitoring']['State']
+        })
+
+    ec = {'AWS.EC2.Instances(val.InstancesId === obj.InstancesId)': response['InstanceMonitorings']}
+    human_readable = tableToMarkdown('AWS EC2 Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def unmonitor_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    data = []
+    response = client.unmonitor_instances(InstanceIds=parse_resource_ids(args.get('instancesIds')))
+
+    for instance in response['InstanceMonitorings']:
+        data.append({
+            'InstanceId': instance['InstanceId'],
+            'MonitoringState': instance['Monitoring']['State']
+        })
+
+    ec = {'AWS.EC2.Instances(val.InstancesId === obj.InstancesId)': response['InstanceMonitorings']}
+    human_readable = tableToMarkdown('AWS EC2 Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def reboot_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.reboot_instances(InstanceIds=parse_resource_ids(args.get('instanceIds')))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Instances were rebooted")
+
+
+def get_password_data_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.get_password_data(InstanceId=args.get('instanceId'))
+    data = {
+        'InstanceId': response['InstanceId'],
+        'PasswordData': response['PasswordData'],
+        'Timestamp': datetime.strftime(response['Timestamp'], '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
this can throw value error. better calculate it before the append and in a try block.